images/modsecurity/rootfs/build.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export NGINX_VERSION=1.20.1
+
+# Check for recent changes: https://github.com/SpiderLabs/ModSecurity-nginx/compare/v1.0.2...master
+export MODSECURITY_VERSION=1.0.2
+
+# Check for recent changes: https://github.com/SpiderLabs/ModSecurity/compare/v3.0.5...v3/master
+export MODSECURITY_LIB_VERSION=v3.0.5
+
+# Check for recent changes: https://github.com/coreruleset/coreruleset/compare/v3.3.2...v3.3/master
+export OWASP_MODSECURITY_CRS_VERSION=v3.3.2
+
+export BUILD_PATH=/tmp/build
+
+#  TODO: Verify and add the same libraries (but not dev) in main container
+apt-get install -y \
```

Comment:
we should run`apt-get update` before this. 

Or execute before this script， like the Dockerfile 👆

Reply:
apt-get has a `--no-install-recommends` flag, we can use it to reduce install packages.

Reply:
Though I was using --no-install-recommends (actually I am, in debian image...)

Will review all again :)

Reply:
Hum I remember now why I didn't added --no-install-recommends here: because modsecurity building is kind of annoying, and we are just using this image to copy the end result (libmodsecurity) to the other image (which is properly a busybox).

Do you think still we should do a --no-install-recommends? About the update I agree, should always update the builder image

Reply:
> Do you think still we should do a --no-install-recommends?

Based on the reasons you mentioned above, I think it is not necessary to add `--no-install-recommends` flag

Reply:
Maybe we can move forward